### DISCUSSION TITLE
Add remote-controlled drones

### DISCUSSION THREAD

**Discussion by MishaUnity** [space-wizards/space-station-14#21862] [Feature Request]
https://github.com/space-wizards/space-station-14/assets/81403616/f6133956-36a5-4d8f-8140-7cd720ac30a5

### _About_

Recently made a prototype of a radio-controlled drone system. How good will they look in the game? 
Drones run on power cells, are piloted from a remote controller, and the functionality can be expanded with modules.

Now the prototype has 6 modules, and 2 types of drones.
> Container Module
> Broadcast Module
> Signal Boost Module
> Acceleration Module
> Stealth Module
> Detonation Module

Basic Radio-controlled Drone
> - Has two module slots

Syndicate Drone
> - Has only one module slot
> - Can pass under airlocks

### _Why?_

It seems to me that this will broaden and deepen the game for the antagonists, add variety to the roboticists.

**Comment by EmoGarbage404** [comment]
This is cool and Id like to see this added as a replacement for the nonfunctional maintenance drones.

**Comment by vanilia123** [comment]
Great work! Can't wait to see more of it

**Comment by juliangiebel** [comment]
That's a really cool feature.

**Comment by nymhole** [comment]
put this in as a PR?
